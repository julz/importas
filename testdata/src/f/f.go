package main

import no_alias_in_config "fmt" // want `import "fmt" has alias "no_alias_in_config" which is not part of config`

func main() {
	no_alias_in_config.Println("test")
}
