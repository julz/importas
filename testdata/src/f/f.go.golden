package main

import "fmt" // want `import "fmt" has alias "no_alias_in_config" which is not part of config`

func main() {
	fmt.Println("test")
}
